test/packages/good_var_migrate_from/_dev/build/docs/README.md

```
@@ -0,0 +1,101 @@
+
+To include a sample event from `sample_event.json`, uncomment and use:
+{{ event "data_stream_name" }}
+
+*/}}
+
+### Inputs used
+{{/* All inputs used by this package will be automatically listed here. */}}
+{{ inputDocs }}
+
+### API usage
+{{/* For integrations that use APIs to collect data, document all the APIs that are used, and link to relevent information */}}
```

markdownlint-cli2 (0.22.1)

[warning] Spaces inside emphasis markers

(MD037, no-space-in-emphasis)